projects.md

```diff
@@ -0,0 +1,70 @@
+# Projects
+
+## Sales Prospect & Internal Proposals
+
+While the Business Planning WG is mainly in charge of sales lead generation,
+all members may pursue leads as they come up.
+These may be a client or a grant opportunity.
+Members are strongly encouraged to record leads into the co-operative's [leads tracking system](https://link.hypha.coop/leads) whether or not we want to pursue this lead.
+
+If a member decides to pursue a **lead**,
+they can reach out to them to see whether there's potential to work together.
+If the lead expresses intention to proceed with a concrete project in mind,
+this becomes a **prospect**.
+The member now heads up this sales effort and should draft an **internal proposal** and present it at a meeting to see whether the co-operative wants to move forward to submit a formal proposal to the prospective client.
```

Comment:
Didn't we talk about working with business planning wg in some fashion before bringing it to everyone.

Also, do you want to pin down as "present it at a meeting", what about time sensitive stuff? Could loomio be appropriate tool here?